static/panes/opt-view.interfaces.ts

```diff
@@ -25,6 +25,11 @@
 export interface OptState {
     optOutput?: OptCodeEntry[];
     source: any; // TODO
+
+    // options/filters
+    'filter-missed': boolean;
```

Comment:
Because this is also part of the API, I would be more comfortable if this were consistent camelCase

Reply:
I don't have a strong opinion, but this doesn't seem to be the current convention for filters in interfaces. Cf eg [ir-view](https://github.com/compiler-explorer/compiler-explorer/blob/3c15839a7702aa84566114f86695bb295d29ebaf/static/panes/ir-view.interfaces.ts#L31-L34) or [opt-pipeline](https://github.com/compiler-explorer/compiler-explorer/blob/3c15839a7702aa84566114f86695bb295d29ebaf/static/panes/opt-pipeline.interfaces.ts#L37-L42).

Reply:
hmm, I guess so, alright, so be it